src/gnu_v2/cplus-dem.c

```diff
@@ -1422,8 +1423,9 @@ demangle_template(struct work_stuff *work, const char **mangled, string *tname,
 	if (is_java_array) {
 		string_append(tname, "[]");
 	} else {
-		if (tname->p[-1] == '>')
-			string_append(tname, " ");
+    // Test cases failing because of this extra space
```

Comment:
Looks like this formatting doesn't match the original one

Reply:
this is a tab to spacing issue I think. This looks fine in my editor
![image](https://github.com/user-attachments/assets/aad85a21-2985-4864-a481-2db947923be2)

Reply:
Just install a plugin/command/whatever to optionally show tabs/spaces, like in my NeoVim, for example:

<img width="748" alt="Screenshot 2024-12-22 at 6 54 41 PM" src="https://github.com/user-attachments/assets/74a7c26e-12b2-4dba-a350-d624af279d64" />

I use this NeoVim config lines for that:
```lua
-- Show tab characters
local tabsshowfn = function()
	vim.o.list = true
	vim.o.listchars = "tab:>-"
end

local tabshidefn = function()
	vim.o.list = false
	vim.o.listchars = ""
end

local tabstogglefn = function()
	if vim.o.list then
		tabshidefn()
	else
		tabsshowfn()
	end
end

vim.keymap.set("n", "<f3>", tabstogglefn)
```

Reply:
Ok 👍